pipeline_dp/accumulator.py

```diff
@@ -174,6 +187,56 @@ def compute_metrics(self) -> float:
 
 
 @dataclass
+class VectorSummationParams:
+    noise_kind: aggregate_params.NoiseKind
+    max_norm: float
+
+_FloatVector = Union[Tuple[float], np.ndarray]
+
+class VectorSummationAccumulator(Accumulator):
+    _vec_sum: np.ndarray
+
+    def __init__(self, params: VectorSummationParams,
+                 values: Iterable[_FloatVector]) -> None:
```

Comment:
As mentioned in a comment below, let's assume that values is List (and then it would be possible to simplify the code)

Reply:
I just noticed that during `add_value` I check for `self._vec_sum is None` - I can easily just loop over `values` without assuming anything except that `values` is an iterable. No need to assume it's a `List`, also I think it's better to leave as `Iterable` as maybe in the future we'd like to do total aggregation over the entire dataset without group-by key.

Reply:
Ok, it makes sense! I agree we don't need to limit ourselves to list w/o necessity.